README.md

```diff
@@ -28,11 +28,11 @@ Rekall is framework for memory forensics.  We leverage Rekall to apply semantic
 
 # Getting Started
 
-You must have compiled and installed the Linux kernel provided in the linux repository within the tenjint project.  Currently, we maintain two branches, v5.2.6-tenjint and rpi-5.2.y-tenjint.  The v5.2.6-tenjint branch builds upon the v5.2.6 Linux kernel tag and the rpi-5.2.y-tenjint branch builds upon the Raspberry Pi rpi-5.2.y branch and runs on the Raspberry Pi 4.  Instructions for building the Linux kernel can be found elsewhere online, but be sure you enable KVM support when configuring your kernel.
```

Comment:
In the meantime, we have updated the default branch to v5.5.8.  Could you include this change while you are updating?  The two branches we maintain are currently v5.5.8-tenjintand rpi-5.2.y-tenjint

Reply:
Done